src/scry/completion/class_module_context.cr

```diff
@@ -0,0 +1,22 @@
+require "./context"
+
+module Scry::Completion
+  class ClassModuleContext < Context
+    def initialize(@text : String, @target : String, @method_db : Completion::MethodDB)
+    end
+
+    def find
+      target_header_index = @target.rindex(":")
+      @method_db.type_match(@target).map do |label|
+        label = label[(target_header_index + 1)..-1] if target_header_index
```

Comment:
just a style comment: At first I didn't saw the `if` at the end of the line, and was wondering 'how the hell does that compile..."

I think that line is too 'complicated' / 'long' for a suffix `if`, and would suggest to make it a multi-line `if`.